apps/researcher/src/messages/en/messages.json

```diff
@@ -109,14 +109,16 @@
     "next": "Next"
   },
   "Filters": {
-    "ownersFilter": "Owners",
     "typesFilter": "Types",
     "subjectsFilter": "Subjects",
     "birthYearsFilter": "Birth years",
     "birthPlacesFilter": "Birth places",
     "deathYearsFilter": "Death years",
     "deathPlacesFilter": "Death places",
     "publishersFilter": "Object data providers",
+    "materialsFilter": "Materials",
+    "creatorsFilter": "Creators",
```

Comment:
Minor: the Object ID standard calls these 'Makers'. Rename the label?

Reply:
This name depends on the filters in the `SearchResult`. The `SearchResult` contains `creators` and not `makers`. Is it possible to change this in the API?

Reply:
Do you mean that the value `Creators` (not the key `creatorsFilter`) depends on something else?

Reply:
Sorry, here you were referring to the value and not the translation key. I will change the value. I first thought you wanted the translation key to be `makersFilter`